src/Twig/GdprExtension.php

```diff
@@ -21,7 +21,7 @@ public function hasPermission(Environment $env, string $permission): bool
     {
         $function = $env->getFunction('sylius_plus_rbac_has_permission');
 
-        if ($function !== null && is_callable($function->getCallable())) {
+        if (is_object($function) && is_callable($function->getCallable())) {
```

Comment:
In my opinion it's pretty weird to check if a variable called "function" is an object or not

Reply:
The method signature has changed between Twig v2 and Twig v3.

Twig v2
```php
    /**
     * @return TwigFunction|false
     */
    public function getFunction($name)
```

In Twig v3
```php
    public function getFunction(string $name): ?TwigFunction
```

So check only with `null` is not enough.